src/main/resources/io/jenkins/plugins/coverage/adapter/CoverageReportAdapterDescriptor/config.jelly

```diff
@@ -10,4 +10,49 @@
             </tr>
         </table>
     </f:entry>
+
+    <f:advanced>
+        <f:entry title="Threshold">
+            <f:repeatable var="threshHold" name="thresholds" items="${instance.thresholds}">
+                <table width="100%">
```

Comment:
Since `Threshold` is describable, I would suggest moving its config to a `config.jelly` resource there an using `f:repeatableProperty` instead. Example: https://github.com/jenkinsci/github-branch-source-plugin/blob/900ea989a624223c88bddb3f1fbb3a3bd2193b5f/src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubConfiguration/config.jelly